### DISCUSSION TITLE
ScriptReference/Physics.IgnoreCollision

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#164] [Notes]
# ScriptReference/Physics.IgnoreCollision

https://docs.unity3d.com/ScriptReference/Physics.IgnoreCollision.html

**Comment by vertxxyz** [comment]
See [`GetIgnoreCollision`](https://docs.unity3d.com/ScriptReference/Physics.GetIgnoreCollision.html) to query whether interactions between two colliders are currently ignored.
